doc/pic/hld_fpmsyncd.md

```diff
@@ -475,6 +515,46 @@ Sample of CONFIG DB snippet given below:
         }
     },
 ```
+#### APPL DB Enhancements  
+
+This feature adds a new NEXT_HOP_GROUP_TABLE, to store next hop group information to be used by one or more routes. Recursive/ecmp routes are represented by referencing a list of nexthop group ids in the nexthop_group field. When this field is used, other fields will not be present.
```

Comment:
"adds a new .."  => "extends the existing .."

Reply:
Done